examples/kafka-authentication-sasl-plain-consumer/kafka_authentication_sasl_plain_consumer.bal

```diff
@@ -0,0 +1,34 @@
+import ballerina/kafka;
+import ballerina/log;
+
+// The `kafka:AuthenticationConfiguration` is used to provide authentication-related details.
+kafka:AuthenticationConfiguration authConfig = {
+    // Provide the authentication mechanism used by the Kafka server.
+    authenticationMechanism: kafka:AUTH_SASL_PLAIN,
```

Comment:
Shall we name it `mechanism`? The word `authentication` is redundant I guess since this is `AuthenticationConfiguration`

Reply:
+1